storage/src/adb/any/fixed/mod.rs

```diff
@@ -387,12 +407,14 @@ impl<
     /// Get the number of operations that have been applied to this db, including those that are not
     /// yet committed.
     pub fn op_count(&self) -> u64 {
-        leaf_pos_to_loc(self.mmr.size()).unwrap()
+        Location::try_from(Position::from(self.mmr.size()))
```

Comment:
There's a new mmr method "leaves()" that you can call here instead, if you don't mind updating.

Reply:
Done